plugins/action/include/plugins/action/action.h

```diff
@@ -106,6 +106,18 @@ class Action : public PluginBase {
      */
     ActionResult return_to_launch() const;
 
+    /**
+     * @brief Send command to reposition the vehicle to a specific WGS84 global position
+     *
+     * This sends the vehicle to a specified lattitude/longitude/altitude coordinates.
+     * @param latitude deg * 1E7
```

Comment:
Why not pass the latitude and longitude as normal coordinates and then multiply by `1E7` on the `ActionResult` code?

Reply:
Possible. I guess that requires to use `double` instead of `float` and then multiply by 1E7, then cast to float when passing to `command.params`, no?

Reply:
lat/long can be entered in degrees now.